recipes/glmnet_py/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "glmnet_py" %}
+{% set version = "v0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/glmnet_python/archive/{{ version }}.tar.gz
+  sha256: ad507d0cc7d8e2194e07b50c420033cf0ce3da2625b6e7da3f8eb2861028f717
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [py2k]
```

Comment:
you can put that just in one line, like:
```
skip: true  # [win or py2k]
```

Reply:
ah I wondered if that was possible - thanks!